Tests iOS/SignOutTests.swift

```diff
@@ -44,4 +44,35 @@ class SignOutTests: XCTestCase {
         try server.stop()
         app.terminate()
     }
+    
+    func test_tappingSignOutshowsLogin() {
+        app.tabBar.settingsButton.wait().tap()
+        tap_SignOut()
+        
+        let account = XCUIApplication()
+        account.alerts["Are you sure?"].scrollViews.otherElements.buttons["Sign Out"].wait().tap()
+        app.accountView.loginText.exists
```

Comment:
This can resolve the `Bool is unused` warning sign.
```suggestion
        XCTAssertTrue(app.accountView.loginText.exists)
```

Reply:
was wondering what that Bool message was about :)